tensorflow/python/platform/self_check.py

```diff
@@ -74,12 +75,22 @@ def preload_check():
         try:
           ctypes.WinDLL(build_info.cudart_dll_name)
         except OSError:
+          with open("cuda_links.json") as f:
```

Comment:
This will almost definitely will not work with bazel.
I am not sure if it would work at all with pip.

So, please avoid an extra file here.

Reply:
Oh I see, I'll just add the dictionary directly to the self_check.py then. Thanks for the headsup.